### PR TITLE
chore(build): update Makefile and CI to adjust release mode

### PR DESCRIPTION
**Pull request by baurine**
Related PD side PR: https://github.com/tikv/pd/pull/7610

To sync the release mode with the TiDB/TiKV/PD, we abandon to use a unique `release` branch to release all versions, and not use `release-version` file to express the version.

### CODE REVIEW

**Comment by baurine** [comment]
@mornyx @shhdgit PTAL, thanks~

**Comment by baurine** [comment]
not merge this PR before the PD side PR is merged.

File: scripts/create_release_tag.js
Comment (wuhuizuo) [inline scripts/create_release_tag.js:36]: It's better throw error to make it exits with a no-zero code.

File: .github/workflows/manual-create-pd-pr.yaml
Comment (baurine) [reply .github/workflows/manual-create-pd-pr.yaml:80]: It seems we don't have the control permission for `tidb-dashboard-bot`, sometimes I want to do some modification for the PR, but we can't do it by using `tidb-dashboard-bot`.

Comment (shhdgit) [reply .github/workflows/manual-create-pd-pr.yaml:80]: get

File: scripts/create_release_tag.js
Comment (baurine) [reply scripts/create_release_tag.js:36]: This script only runs in local, so it won't break CI flows.

File: Makefile
Comment (mornyx) [inline Makefile:18]: https://github.com/pingcap/tidb-dashboard/pull/1627/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L40

The line above can be removed since we use git tag as `RELEASE_VERSION` and `-fips` will appear in git tag.

Comment (baurine) [reply Makefile:18]: ok, let me check.

**Review by mornyx — APPROVED** [review]

**Review by shhdgit — APPROVED** [review]
lgtm.

**Pull request merged**
